terraform/cloudfoundry/cf_rds.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_db_instance" "cf" {
   allocated_storage    = 10
   engine               = "postgres"
   engine_version       = "9.5.4"
-  instance_class       = "db.t2.small"
+  instance_class       = "db.m4.large"
```

Comment:
> Not sure what is the point of trying to save costs with this DB.

Could you explain why?

> I wouldn't use a credit based instance type. If there is any issue related to the DB it will eventually run out of credits and we will be in the same situation.

I *think* @saliceti might have meant that the CPU and RAM specifications of `t2.medium` were sufficient, were it not throttled. So `m3.medium` would meet the same needs without being throttled.